test/test_cuda.py

```diff
@@ -328,81 +345,108 @@ def tmp(t):
     ('kthvalue', small_3d_unique, lambda t: [3],),
     ('kthvalue', small_3d_unique, lambda t: [3, 1], 'dim'),
     ('kthvalue', small_3d_unique, lambda t: [3, -1], 'neg_dim'),
-    ('lerp', small_3d, lambda t: [small_3d(t), 0.3],),
-    ('max', small_3d_unique, lambda t: [],),
-    ('max', small_3d_unique, lambda t: [1], 'dim'),
-    ('max', small_3d_unique, lambda t: [-1], 'neg_dim'),
+    ('lerp', small_3d, lambda t: [small_3d(t), 0.3], '', types, False, "skipIfRocm:HalfTensor"),
+    ('max', small_3d_unique, lambda t: [], '', types, False, "skipIfRocm:HalfTensor"),
+    ('max', small_3d_unique, lambda t: [1], 'dim', types, False,
+        "skipIfRocm:ByteTensor,CharTensor,DoubleTensor,FloatTensor,HalfTensor,IntTensor,LongTensor,ShortTensor"),
```

Comment:
This one is particular strange. Aren't you just trying to disable the ROCm test entirely?

Reply:
Agreed. This made sense when we started on this effort because we didn't want to unnecessarily disable all datatypes for a test if some of them worked. However, now that we're done with that effort, we can simply replace all examples like the above with `skipIfRocm`

Reply:
@jithunnair-amd please send a PR :-)

Reply:
Will do in a short while. I'm working with @lcskrishna to see how many distinct groupings we have and whether it'd make sense to group the types.